samples/wifi/raw_tx_packet/README.rst

```diff
@@ -0,0 +1,156 @@
+
+.. options-from-kconfig::
+
+Configuration options for Connected mode
+
+To configure the sample in connected mode, you must configure the following Wi-Fi® credentials in the :file:`prj.conf` file:
+
+* ``Service Set Identifier (SSID)``: Defines the name of your Wi-Fi network.
+* ``Key management``: Specifies the key management method, such as WPA2, WPA3, or other encryption protocols.
+* ``Password``: Sets the Pre-shared Key (PSK) of your Wi-Fi network.
+
+.. note::
+   You can also use ``menuconfig`` to enable ``Key management`` option.
+
+See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
+
+Configuration options for Idle mode
+
+To configure the sample in Idle mode, you must configure the following parameter in the :file:`prj.conf` file:
+
+* ``Channel``: Specifies the Wi-Fi channel to be used for communication on the wireless network.
+
+Configuration options for Raw TX Packet header
+
+* ``Rate value``: Specifies the data transmission rate.
+* ``Rate flags``: Specifies the data transmission mode.
+* ``Queue number``: Specifies the transmission queue to which raw TX packets are assigned for sending.
+
+Additionally, you must configure the following option in the :file:`prj.conf` file: to define the time delay between raw TX packets:
+
+``Inter-frame delay``: Sets the time duration between raw TX packets.
+
+IP addressing
+*************
+The sample uses DHCP to obtain an IP address for the Wi-Fi interface.
+It starts with a default static IP address to handle networks without DHCP servers, or if the DHCP server is not available.
+Successful DHCP handshake will override the default static IP configuration.
+
+You can change the following default static configuration in the :file:`prj.conf` file:
+
+.. code-block:: console
+
+  CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.168.1.98"
+  CONFIG_NET_CONFIG_MY_IPV4_NETMASK="255.255.255.0"
+  CONFIG_NET_CONFIG_MY_IPV4_GW="192.168.1.1"
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/raw_tx_packet`
+
+.. include:: /includes/build_and_run_ns.txt
+
+The sample can be built for the following configurations:
+
+* Continuous transmission with the Station-connected mode.
+* Fixed number of packet transmission with the Station-connected mode.
+* Continuous transmission with the Station-idle mode.
+* Fixed number of packet transmission with the Station-idle mode.
+
+To build for the nRF7002 DK, use the ``nrf7002dk_nrf5340_cpuapp`` build target.
+The following are examples of the CLI commands:
+
+* Continuous transmission with the station-connected mode:
+
+  .. code-block:: console
+
+     west build -b nrf7002dk_nrf5340_cpuapp -- -DCONFIG_RAW_TX_PACKET_SAMPLE_CONNECTION_MODE=y -DCONFIG_RAW_TX_PACKET_SAMPLE_PACKET_TRANSMISSION_MODE=0
+
+* Fixed number of packet transmission with the station-connected mode:
```

Comment:
```suggestion
* Fixed number of packet transmission with the Station-connected mode:
```

Reply:
This got missed and at lines 108, 114 as well.